.circleci/config.yml

```diff
@@ -10,10 +10,10 @@ commands:
     steps:
       - run:
           name: Setup Rust
-          command: |
-            rustup install stable
```

Comment:
Were these commands commented out just because there was a temporary mismatch between the Rust version specified in the Dockerfile and the Rust version being pulled in here? If so, we can probably the Dockerfile and uncomment these lines now, right? Looks like there's a 1.61 buster image now: https://hub.docker.com/_/rust

Reply:
Alternatively, do we need these commands at all if we always want to use the Rust version given to us by the Docker image?

Reply:
FWIW, Rust Docker images usually are available the day after a rust release. It's been quite some time since there's been a breaking change between versions of rust, or a rust update causing something like `fmt` or `clippy` to fail. I'd be curious why those were commented out.